resubmit.py

```diff
@@ -49,33 +50,30 @@ def modifySchema(helper, workflow, user, group, cache, events, firstLumi, backfi
     paramBlacklist = ['BlockCloseMaxEvents', 'BlockCloseMaxFiles', 'BlockCloseMaxSize', 'BlockCloseMaxWaitTime',
```

Comment:
a few more entries need to be added in this list.
DN, AutoApproveSubscriptionSites, NonCustodialSites, ...